integreat_cms/cms/views/media/media_actions.py

```diff
@@ -81,12 +81,17 @@ def get_directory_content_ajax(request, region_slug=None):
     directory = None
     if request.GET.get("directory"):
         directory = get_object_or_404(
-            Directory.objects.filter(Q(region=region) | Q(region__isnull=True)),
+            Directory.objects.filter(
+                Q(region=region) | Q(region__isnull=True),
+                Q(region=region, is_hidden=False) | Q(region__isnull=True),
```

Comment:
```suggestion
                Q(region=region, is_hidden=False) | Q(region__isnull=True),
```
The second line is a more restrained filter than the first line, so we can just remove the first one and get the same results.